core/src/test/java/com/opensymphony/xwork2/ognl/accessor/XWorkListPropertyAccessorTest.java

```diff
@@ -85,7 +85,7 @@ public void testAutoGrowthCollectionLimit() {
 
     public void testDeprecatedAutoGrowCollectionLimit() {
```

Comment:
With the deprecated `setDeprecatedAutoGrowCollectionLimit()` being dropped, I think `testDeprecatedAutoGrowCollectionLimit()` now becomes a duplicate of `testAutoGrowthCollectionLimit()`.
If that is the case, then you might be able to drop `setDeprecatedAutoGrowCollectionLimit()` completely, if you want.

Reply:
Good catch, dropped!